### DISCUSSION TITLE
How do you escape autolink?

### DISCUSSION THREAD

**Discussion by patricktokeeffe** [community/community#123033] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there any way to escape the autolink function without resorting to weird trickery like zero-width characters or hidden spans?

| Raw entry | Rendered Output | |
|-|-|-|
| `#1` | #1 | correct |
| `\#1` | \#1  | escape character ignored |
| `\\#1` | \\#1 | ignored, sort of |
| `&num;1` | &num;1 | failed workaround |
| `\&num;1` | \&num;1 | double failure |
| `&#35;1` | &#35;1 | failed workaround |
| `\&#35;1` | \&#35;1 | wow |
| `#&zwnj;1` | #&zwnj;1 | zero-width non-joiner ([ref](https://gist.github.com/alexpeattie/4729247?permalink_comment_id=5026040#gistcomment-5026040)) |
| `#<span/>1` | #<span/>1 | another workaround ([ref](https://gist.github.com/alexpeattie/4729247?permalink_comment_id=3221445#gistcomment-3221445)) |

**Comment by Doruk34** [comment]
This works for me :

http://<i></i>example.com

which gives

http://example.com

instead of

http://example.com/

And it involves less, easier typing :)
